app/models/form526_submission.rb

```diff
@@ -46,14 +46,14 @@ def auth_headers
     @auth_headers_hash ||= JSON.parse(auth_headers_json)
   end
 
-  def perform_ancillary_jobs_handler(status, options)
+  def perform_ancillary_jobs_handler(_status, options)
```

Comment:
if `status` isn't being used in this method anymore, why are we keeping it around as an argument?

Reply:
This is a relic from the call back. Call back methods will always receive `status` and `options`.

Reply:
ahh thanks for the clarification